docs/src/api/class-locatorassertions.md

```diff
@@ -2180,7 +2198,7 @@ Expected options currently selected.
 * since: v1.23
 
 
-## async method: LocatorAssertions.toMatchAriaSnapshot#1
+## async method: LocatorAssertions.toMatchAriaSnapshot
```

Comment:
Looks like our practise which we before used was to not explicitly call the first overload `#1` - this would have required changes in the Python generator - lets instead keep using the previous practise?